### ISSUE TITLE
[uix] hovering both lazers hover over a button and clicking causes the button to rapid fire.

### ISSUE THREAD

**Issue by mralext20** [Frooxius/NeosPublic#774]
this is annoying, especially with uix based keyboards

**Comment by H3BO3** [comment]
This is still an issue, I thought @shiftyscales opened this same thing a while back. It may be a duplicate if so

**Comment by H3BO3** [comment]
I can see the issue for requesting some kind of native rapid-click functionality, but can't find much else. This still seems to be a prevalent bug and it would be nice if it could finally be fixed.

**Comment by shiftyscales** [comment]
Technically it was already reported via this related issue: #636

**Comment by TehTurk** [comment]
See this is more of a bug feature then a bug. Alot of us would actually be somewhat upset from this being removed. So if we do, I'd like there to be some sort of easier rapid clicking in some form.

**Comment by H3BO3** [comment]
It's hardly a bug report, froox may have addressed the feature but the bug was never acknowledged or fixed

**Comment by shiftyscales** [comment]
It wasn't fixed because there were users that expressed the desire to keep that functionality as a mechanic in some form.

So at this point, they are linked/related issues. This bug could be fixed alongside introducing that mechanism in some form.

**Comment by TehTurk** [comment]
Idk having something to affect the event speed? But the only way this can trigger if your holding on both triggers and hover over the same button. While we can design stuff to replace this elegantly, sometimes inelegant stuff just works well. This is only an issue for UIX canvases as well, because activating this actually takes effort vs not.

**Comment by H3BO3** [comment]
No, it doesn't. This happens to me often. I can't type quickly because triggers on each controller end up intersecting and it spams a bunch of presses. If it was explicitly the same button, fine, but it seems to happen on the same canvas, the keyboard especially. If there was a delay period before it engaged that could work too.

**Comment by TehTurk** [comment]
Hmm see this is never an issue because I've configured my controllers to do a tap typing vs holding down the trigger. Delay would be reasonable but it seems weird this is an issue now, this has been around for awhile.

**Comment by H3BO3** [comment]
not an excuse, it's a bug that pops up every now and then and mildly inconveniences me. I'm finally addressing it

**Comment by TehTurk** [comment]
Understandable. I just don't want to see this fixed until there is a proper alternative. Your suggestion in the original thread would be apt tho. The thing is if your using my bindings tho h3bo, you wana change the trigger then because it requires 2 clicks already for it to operate I thought.

**Comment by Hayden-da-Serg** [comment]
That's interesting @H3BO3 I've only experienced this with single buttons, not the same canvas, are you sure you're not just holding the trigger for too long? I just tested and the keyboard repeats input even if I'm just using one controller (At least the default one with slight modifications)

**Comment by H3BO3** [comment]
Perhaps I'm remembering it wrong. Take that one with a grain of salt then - Are you saying it spams keys when you're only using one trigger?

**Comment by Hayden-da-Serg** [comment]
Yup

**Comment by H3BO3** [comment]
yeah, I think that's what I've seen. Either way I can only use one laser or there's more and more chance of it happening. If this is formalized it should require both triggers

**Comment by mralext20** [comment]
i agree, this *can* be maintained, as long as it requires both triggers. 

that would turn this 'bug' into an actual feature of neos, allowing the previously mentioned #636 to be closed aswell.
